dist/index.js

```diff
@@ -371,7 +371,7 @@ var _initialiseProps = function _initialiseProps() {
       payload = _this3.props.payloadModifier instanceof Function ? _this3.props.payloadModifier(payload) : _this3.props.payloadModifier instanceof Object ? Object.assign(payload, _this3.props.payloadModifier) : payload;
     }
 
-    _this3.xhr.open('POST', (customEndpoint && customEndpoint.slice(-1) === '/' ? customEndpoint.slice(0, -1) : customEndpoint || 'https://suggestions.dadata.ru') + '/suggestions/api/4_1/rs/suggest/' + type);
+    _this3.xhr.open('POST', (customEndpoint && customEndpoint.slice(-1) === '/' ? customEndpoint.slice(0, -1) : customEndpoint || 'https://suggestions.dadata.ru/suggestions/api/4_1/rs/suggest') + '/' + type);
```

Comment:
I think that i understood the problem, but this - changed nothing, logic of string concatination are equal, at before to after your change's